braindecode/preprocessing/preprocess.py

```diff
@@ -153,7 +153,9 @@ def preprocess(
 
     parallel_processing = (n_jobs is not None) and (n_jobs != 1)
 
-    list_of_ds = Parallel(n_jobs=n_jobs)(
+    import platform
```

Comment:
can you move this platform import to above?

Reply:
I have also moved the import statement to the top.